### PR TITLE
[Fix] Minor fixes

### PR DESCRIPTION
**Pull request by s0lus**

### CODE REVIEW

File: src/Command/Check.php [hunk 1]
```diff
@@ -36,7 +36,7 @@ function ($errno, $errstr, $errfile, $errline, $errcontext) use ($output) {
         );
 
         /** @var \Swagger\Annotations\Swagger $swagger */
-        $swagger = $swaggerSerializer->deserialize(
```
Comment (s0lus) [inline src/Command/Check.php]: @ovr my IDE tells me, that variable is not used

Comment (ovr) [reply src/Command/Check.php]: @s0lus but why you didnt remove php docblock?)

Comment (s0lus) [reply src/Command/Check.php]: @ovr Updated!

**Comment by ovr** [comment]
Thank you ❤️

**Pull request merged**